Tests/Editor/Deployment/DeploymentSettingsTests.cs

```diff
@@ -118,18 +100,11 @@ public void BuildFilePath_WhenFormFilledAndRestore_IsExpected(bool coreSuccess)
             string testBuildExePath = "test path" + id;
             string testBuildExePath1 = "test path1" + id;
 
-            var coreApiMock = new Mock<CoreApi>();
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.DeploymentBuildFilePath, coreSuccess, testBuildExePath1);
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.DeploymentGameName, false, null);
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.DeploymentScenarioIndex, false, null);
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.DeploymentBuildFolderPath, false, null);
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.LaunchParameters, false, null);
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.BuildOperatingSystem, false, null);
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.FleetName, false, null);
-            coreApiMock.SetUpCoreApiWithSetting(SettingsKeys.BuildName, false, null);
-
+            var stateManagerMock = new Mock<StateManager>();
+            stateManagerMock.SetupProperty(state => state.DeploymentBuildFilePath,
+                coreSuccess ? testBuildExePath1 : null);
```

Comment:
It's not clear what we're actually testing here. We mock state manager to return a value, and then we set a folder path and expect the exe path matches the mock? It looks like we're just testing that we mocked a value. The same goes for the other tests changed here

Reply:
Ack, I'm going to leave this open then so we don't forget this for unit test changes